### PR TITLE
RDAPI-25047 - Updated release notes for Cassandra Upgrade, ICAP Message Preview and APIM Unauthenticated req rate limiter

### PR DESCRIPTION
**Pull request by credmond-axway**
Updated release notes for Cassandra Upgrade, ICAP Message Preview and APIM Unauthenticated req rate limiter

## Deploy preview link

https://deploy-preview-2075--axway-open-docs.netlify.app/docs/apim_relnotes/20210830_apimgr_relnotes/
https://deploy-preview-2075--axway-open-docs.netlify.app/docs/apim_reference/system_props/
https://deploy-preview-2075--axway-open-docs.netlify.app/docs/apim_policydev/apigw_external_connections/common_icap_conf/

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by dshielaxway — APPROVED** [review]
Looks good @credmond-axway. Thanks!

**Review by sgoka-axway — APPROVED** [review]
Looks good!

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md [hunk 1]
```diff
@@ -217,36 +235,6 @@ These new properties were added in JAXP 1.5 specification, which is supported by
 
 Related Issue: RDAPI-22218
 
-### No VAPI matched request after upgrade from 7.5.3 SP12 using inbound security policy
```
Comment (kmercierwhite) [inline content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:220]: Why is this deleted?

Comment (credmond-axway) [reply content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:220]: I believe it is because this issue has now been fixed.

@sgoka-axway Can you confirm?

Comment (sgoka-axway) [reply content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:220]: @kmercierwhite , we have fixed this issue in the last sprint RDAPI-22086

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md [hunk 2]
```diff
@@ -46,6 +55,16 @@ It will still be possible to deploy upgraded configurations containing the retir
 
 The cadence of the updates for API Gateway, API Manager, and API Portal has changed. From the August update onwards, the update schedule follows a three months release cadency
 
+### Apache Cassandra Upgrade
+
+In this update, Apache Cassandra has been upgraded to version 3.11.11. For installation instructions, see [Install an Apache Cassandra database](apim_installation/apigtw_install/cassandra_install/). For details on how to upgrade exisiting Apache Cassandra environments, see [Upgrade Apache Cassandra](/docs/apim_installation/apigw_upgrade/upgrade_cassandra/).
```
Comment (kmercierwhite) [inline content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:60]: Saying Apache Cassandra has been upgraded is a bit misleading. We should instead say

In this update, API Gateway now supports Apache Cassandra 3.11.11.

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md (hunk 1 above)
Comment (kmercierwhite) [reply content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:220]: Thanks!

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md (hunk 2 above)
Comment (credmond-axway) [reply content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md]: @kmercierwhite updated the Cassandra upgrade wording

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md [hunk 3]
```diff
@@ -30,6 +30,15 @@ The Elastic-Stack (Elasticsearch, Logstash, and Kibana) solution is now producti
 
 It is now possible to deploy YAML configuration stores in the API Gateway Manager web UI on port 8090. See [Manage domain topology in API Gateway Manager](/docs/apim_administration/apigtw_admin/managetopology/#deploy-a-yaml-deployment-package)
 
+### ICAP Message Preview now supported; RFC 3507 Section 4.5
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:33]: (RFC 3507 Section 4.5) - please remove this from the heading. If it's super important and helpful to customers, you can add it to the rext. But is it?

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md [hunk 4]
```diff
@@ -30,6 +30,15 @@ The Elastic-Stack (Elasticsearch, Logstash, and Kibana) solution is now producti
...
+
+It is now possible to enable Message Preview by configuring the 'Preview' custom ICAP header. See [Custom headers in Configure ICAP servers](/docs/apim_policydev/apigw_external_connections/common_icap_conf/#custom-headers)
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:35]: The **custom-headers** page/section says '_You can configure custom headers in the Advanced tab for the REQMOD and RESPMOD options..._', but here you say that you can do it by '_configuring the 'Preview' custom ICAP header_'. This is confusing. I'd suggest that in the Release Notes (RN) you just announce the new feature, and leave the explanation on how to configure it to the actual documentation - https://axway-open-docs.netlify.app/docs/apim_policydev/apigw_external_connections/common_icap_conf/#custom-headers

The next paragraph (_Setting a preview length notifies the ICAP server..._) also seems to be an info that customers need to see in the docs, which they'll read every time they need to configure this setting, not just once, here in the RN.

Can you make these changes please?

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md [hunk 5]
```diff
@@ -46,6 +55,16 @@ It will still be possible to deploy upgraded configurations containing the retir
 
 The cadence of the updates for API Gateway, API Manager, and API Portal has changed. From the August update onwards, the update schedule follows a three months release cadency
 
+### Apache Cassandra Upgrade
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:58]: (In this update) -> Unnecessary, please remove it.
(apim_installation/apigtw_install/cassandra_install/) -> broken link

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md [hunk 6]
```diff
@@ -46,6 +55,16 @@ It will still be possible to deploy upgraded configurations containing the retir
...
+
+In this update, API Gateway now supports Apache Cassandra 3.11.11. For installation instructions, see [Install an Apache Cassandra database](apim_installation/apigtw_install/cassandra_install/). For details on how to upgrade exisiting Apache Cassandra environments, see [Upgrade Apache Cassandra](/docs/apim_installation/apigw_upgrade/upgrade_cassandra/).
+
+### API Manager Unauthenticated request rate limiter
+
+The rate limiter monitors the number of requests that a user can send to API Manager without an active session. If the number of requests exceeds the configured limit, a 403 Forbidden error response is returned, and the IP address of the client is locked for the configured time.
+
+For more information, see [Configure API Manager unauthenticated request rate limiter](/docs/apim_administration/apimgr_admin/api_mgmt_config).
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:66]: don't you prefer to link to the specific section?
 /docs/apim_administration/apimgr_admin/api_mgmt_config/#configure-api-manager-unauthenticated-request-rate-limiter

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md [hunk 7]
```diff
@@ -46,6 +55,16 @@ It will still be possible to deploy upgraded configurations containing the retir
 
 The cadence of the updates for API Gateway, API Manager, and API Portal has changed. From the August update onwards, the update schedule follows a three months release cadency
 
+### Apache Cassandra Upgrade
+
+In this update, API Gateway now supports Apache Cassandra 3.11.11. For installation instructions, see [Install an Apache Cassandra database](apim_installation/apigtw_install/cassandra_install/). For details on how to upgrade exisiting Apache Cassandra environments, see [Upgrade Apache Cassandra](/docs/apim_installation/apigw_upgrade/upgrade_cassandra/).
+
+### API Manager Unauthenticated request rate limiter
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:62]: For the RN, we need something more "markety", not the same text of the docs. e.g., 

## Request rate limiter is available in API Manager
You can now configure a rate limiter in your API Manager to <avoid this or that issue and improve performance> ??
For more information, see...

@kmercierwhite

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md (hunk 6 above)
Comment (sgoka-axway) [reply content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:66]: Yes, we can link to the specific section.ty!

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md (hunk 7 above)
Comment (sgoka-axway) [reply content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:62]: In that case , then

**Unauthenticated request rate limiter is available in API Manager**

You can now configure an unauthenticated request rate limiter in your API Manager.
 For more information, see [Configure API Manager unauthenticated request rate limiter](/docs/apim_administration/apimgr_admin/api_mgmt_config/#configure-api-manager-unauthenticated-request-rate-limiter).

@kmercierwhite

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md (hunk 4 above)
Comment (dshielaxway) [reply content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:35]: Preview is one of the custom headers. I mention this in the first sentence: _It is now possible to enable Message Preview by configuring the **'Preview' custom ICAP header**_.

I'll rearrange.

File: content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md (hunk 3 above)
Comment (dshielaxway) [reply content/en/docs/apim_relnotes/20210830_apimgr_relnotes.md:33]: We've implemented RFC 3507 Section 4.5, so it is helpful that customers know where to get more details about the change in behaviour, and the benefits of using Message Preview.

I'll rearrange the section

File: content/en/docs/apim_policydev/apigw_external_connections/common_icap_conf.md [hunk 8]
```diff
@@ -71,3 +71,13 @@ You can configure custom headers in the **Advanced** tab for the above **REQMOD*
 When the **ICAP** filter is executed, the specified custom headers are added to outgoing **REQMOD** and **RESPMOD** requests.
 
 Custom headers override default headers with the same name, so if a custom header has the same name as a header that is automatically added by the filter implementation, only the custom header value is sent.
+
+#### Enable message preview in request headers
```
Comment (andreamussap) [inline content/en/docs/apim_policydev/apigw_external_connections/common_icap_conf.md:75]: @credmond-axway I've made some changes to this section. Please let me know if the technical information is accurate and good to go?

Comment (credmond-axway) [reply content/en/docs/apim_policydev/apigw_external_connections/common_icap_conf.md:75]: + @dshielaxway

Comment (dshielaxway) [reply content/en/docs/apim_policydev/apigw_external_connections/common_icap_conf.md:75]: I think the heading of the section should be **Message Preview**. That is the name of the feature, and what it is referred to in the RFC.

The text '**To enable Message Preview, add and configure the 'Preview' custom header**' should probably be there too to  make it clear that is how the feature is enabled.

> Previewing a message can yield performance improvements for certain use cases, in particular where large payloads are being sent. Also, virus checkers can usually certify payloads as being ‘clean’ simply by looking at the first few bytes.

I think the original text read better as the '**virus checkers can usually certify payloads as being ‘clean’ simply by looking at the first few bytes**' was intended as an aside to the preceding sentence, hence the ';'. But it's very minor. I'll leave it up to you.

I also noticed that the CSS renders the 'Message Preview' heading the same size as the 'Custom Headers' heading despite it being a sub-heading of that section. Again, very minor.

Other than that, looks good to me.
